app/helpers/shift_commerce/cache_helper.rb

```diff
@@ -5,7 +5,8 @@ def shift_cache object, *args
       cache(shift_cache_key object, *args) { yield }
     end
 
-    def menus_cache(reference, banner_reference = nil, dependent_reference = nil, options = {})
+    def menus_cache(reference, cache_version = nil, banner_reference = nil, dependent_reference = nil, options = {})
```

Comment:
If we pull in from the Menus service constants then we don't need to pass it in
```suggestion
    def menus_cache(reference, banner_reference = nil, dependent_reference = nil, options = {})
```